package-files/systemd/infra.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Modern infrastructure access.
+Documentation=https://docs.infrahq.com
+Wants=network-online.service
+After=network-online.service
+ConditionPathExists=/etc/infra/config.yaml
```

Comment:
I think we should use `/etc/infra/connector.yaml`. The `infra` binary can run in different modes, (cli, server, connector, agent) and we probably want a different config file for each.